go-controller/pkg/ovn/logical_switch_manager/logical_switch_manager.go

```diff
@@ -286,26 +287,51 @@ func (manager *LogicalSwitchManager) AllocateNextIPs(switchName string) ([]*net.
 	return ipnets, nil
 }
 
+func ipNetSlicesEqual(allocated, potential []*net.IPNet) bool {
```

Comment:
i dont think we need to compare or store the DRIPs in the lsm. The source of truth is the annotation. We should allocate all pre-existing DRIPs and pod ips and not add any new ones until that is complete.

Reply:
done